flow/workflows/snapshot_flow.go

```diff
@@ -196,8 +193,8 @@ func (s *SnapshotFlowExecution) cloneTables(
 	slotInfo *protos.SetupReplicationOutput,
 	maxParallelClones int,
 ) {
-	logrus.Infof("cloning tables for slot name %s and snapshotName %s",
-		slotInfo.SlotName, slotInfo.SnapshotName)
+	slog.Info(fmt.Sprintf("cloning tables for slot name %s and snapshotName %s",
```

Comment:
`workflow.Context` no good for `InfoContext`?

Reply:
Nope. The only time passing `workflow.Context` to children works is when giving it to `ExecuteActivity()` (which I think makes it compatible with `context.Context` internally). Otherwise the types are not the same and `context.WithValue(aWorkflowContext, ..,...)` will throw a type error.